### ISSUE TITLE
[4.0] backend dashboard

### ISSUE THREAD

**Issue by micker** [joomla/joomla-cms#25579]
### What needs to be fixed
new dashboard add 2 buttons 
1 for list
1 for add
but i think the design isn't good because too many button in same zone, different size, different design ...
![2019-07-16_09h24_10](https://user-images.githubusercontent.com/2539440/61274152-7a9f0980-a7ab-11e9-8e6d-802c99c535b4.png)
### Why this should be fixed
Maybe when have more consistent design to use same systeme like media manager

![2019-07-16_09h28_40](https://user-images.githubusercontent.com/2539440/61274455-247e9600-a7ac-11e9-9efa-05305a19987b.png)

Need to check if accessibility is good ..

**Comment by brianteeman** [comment]
the accessibility cant be any more broken than the current code

**Comment by micker** [comment]
=> i do it with simple css modfication (i am not great css writter i didn't push code) no html change ..
for me have 2 same labels for each button isn't good for acessibility. ex users button to user list and an other button user to add isn't consistent..
its a detail but its the first screen for every user
maybe this can be better
![2019-07-16_11h02_04](https://user-images.githubusercontent.com/2539440/61281007-2d299900-a7b9-11e9-8430-8077e644613c.png)
@coolcat-creations @bembelimen  what do you think ?

**Comment by coolcat-creations** [comment]
@micker no idea. the design proposal looked like this - I am currently not involved in the further process,...

![grafik](https://user-images.githubusercontent.com/828371/61282765-310aea80-a7bc-11e9-8914-a5fd7def14b5.png)

**Comment by micker** [comment]
@coolcat-creations cool the number in button ! that usefull and more clear !
why this difference ?

**Comment by brianteeman** [comment]
@micker the numbers are enabled/disabled in the admin modules

**Comment by chmst** [comment]
@micker please read this https://github.com/joomla/backend-template/issues/115 and related items. 

Abuot the number of icons on a panel: You switch on and off the icons with parameters, you can show more or less modules with icons.

**Comment by micker** [comment]
ok thanks for all reply !

**Comment by ghost** [comment]
@micker please close if the open Issue isn't needed.

**Comment by alikon** [comment]
@micker 
plus if this and the other's refers to #25570 wich is not merged yet please comment on the pr's and not  open issues  for not merged code
 thanks for your attention

**Comment by micker** [comment]
for number its ok
but for me name and design for add button need to be more compact and accessible
issue need to be debat at first not ?

**Comment by micker** [comment]
@alikon sorry same post in same time

**Comment by alikon** [comment]
but if your issues are related to #25570 it is better to have all in one place

**Comment by ghost** [comment]
@micker please comment this Issue too on #25570 so we have discussion at on place and can close this one.

**Comment by alikon** [comment]
same for #25582

**Comment by alikon** [comment]
thanks @micker
